### PR TITLE
Document getting available explicit and service permissions

### PR DESCRIPTION
**Pull request by ivaylomitrev**
This commit adds documentation about 1 new web service:
* GET noark5/v1/access-group/info

### CODE REVIEW

**Comment by ivaylomitrev** [comment]
The PR is targeted towards feature/add-documentation-about-managing-access-modifiers as they both modify the same file at the same place.

**Review by orlinDocumaster — APPROVED** [review]

**Review by vulkodava — APPROVED** [review]

File: version1/04 - Web services.md [hunk 1]
```diff
@@ -1113,6 +1113,57 @@ Content-Type: application/json
...
+* View changelog
+* Journal
+* Store documents
+* GUI administrator
+* Security administrator
+
+New service permissions may be added over time.
+
+The available explicit permissions at the time of writing this documentation are:
+* ReadThis
```
Comment (teodoradobreva) [inline version1/04 - Web services.md:1155]: Is there any additional documentation about what these explicit permissions mean? Same about the service permissions.

Comment (ivaylomitrev) [reply version1/04 - Web services.md:1155]: No, not right now.

I'll add a description of this in a separate PR, if you don't mind?

**Review by teodoradobreva — APPROVED** [review]

**Comment by ivaylomitrev** [comment]
Changed the target of the PR (and squashed) after PR#43 was merged.

**Pull request merged**